schemas/staff_hours.schema.json

```diff
@@ -0,0 +1,150 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://ocf.berkeley.edu/schemas/staff_hours.schema.json",
+  "definitions": {
+    "time": {
+      "type": "string",
+      "pattern": "^[012]?[0-9]:[0-9][0-9]$"
+    },
+    "timerange": {
+      "type": "array",
+      "items": [
+        {
+          "$ref": "#/definitions/time"
+        },
+        {
+          "$ref": "#/definitions/time"
+        }
+      ]
+    },
+    "staff_hours_entry": {
+      "type": "object",
+      "properties": {
+        "time": {
+          "$ref": "#/definitions/timerange"
+        },
+        "staff": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "minItems": 1,
+          "uniqueItems": true
+        }
+      },
+      "required": [
+        "time",
+        "staff"
+      ],
+      "additionalProperties": false
+    },
+    "staff_position": {
+      "type": "object",
+      "properties": {
+        "username": {
+          "type": "string"
+        },
+        "position": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "username",
+        "position"
+      ],
+      "additionalProperties": false
+    }
+  },
+  "type": "object",
+  "properties": {
+    "staff-hours": {
+      "type": "object",
+      "properties": {
+        "Monday": {
+          "type": [
+            "array",
+            "null"
+          ],
+          "items": {
+            "$ref": "#/definitions/staff_hours_entry"
+          }
```

Comment:
Can we make a type for staff_hours_day or staff_hours_entry_list or something?

Reply:
Do you mean having the days as an array instead of hardcoded properties?

Reply:
This can be made more concise, see https://github.com/ocf/etc/blob/master/schemas/hours.schema.json#L79-L109